specification/nhais-adaptor.yaml

```diff
@@ -285,40 +285,99 @@ paths:
         ## Overview
         Use this endpoint to send an Amendment to NHAIS.
 
-        This is a 'patch' operation, meaning you can update specific parts of the patient record (such as a name or an address), as opposed to having to update the entire record.
+        This is a 'patch' operation, meaning you can update specific parts of the patient record (such as a name or an address),
+        as opposed to having to update the entire record.
 
-        ## Operation Id
-        When you submit an amendment an operation id will be included in the response. This operation id MUST be retained
-        to match against the asynchronous reply from NHAIS
+        The adaptor is not a FHIR server and cannot perform actual patches onto Patient resources. Instead the patches are
+        translated into an equivalent EDIFACT message. This places some restrictions on the patches accepted by the adaptor:
 
-        ## Payload Description
+        * The value of "path" must match exactly those listed in the "FHIR / JSONPatch Field Mappings" table below. Logically
+          equivalent alternatives are not permitted.
+        * The only supported operations are add, replace, and remove
+        * The add and replace operations are synonymous for GP Links
+        * The remove operation MUST only be used for data items for which GP Links permits removal
```

Comment:
Do we validate that in NHAIS? If not, what will happen if one would try to remove protected field?